### PR TITLE
Ethos ecm 60

### PR DESCRIPTION
**Pull request by btensay**
Code change added to implement ecm-60.

https://tools.hmcts.net/jira/browse/ECM-60

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java [hunk 1]
```diff
@@ -33,8 +33,8 @@
 import java.util.Map;
 
 import static com.google.common.base.Strings.isNullOrEmpty;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.*;
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java:36]: Checkstyle

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java [hunk 2]
```diff
@@ -366,7 +367,12 @@ private static StringBuilder getCaseCauseListByRoom(ListingData listingData, Str
     public static StringBuilder getListingDate(ListingData listingData) {
         var sb = new StringBuilder();
         if (listingData.getHearingDateType() != null
-                && listingData.getHearingDateType().equals(RANGE_HEARING_DATE_TYPE)) {
+                && RANGE_HEARING_DATE_TYPE.equals(listingData.getHearingDateType())
+                && SERVING_CLAIMS_REPORT.equals(listingData.getReportType())) {
+            return getServedClaimsReportPeriod(listingData);
+
+        } else if (listingData.getHearingDateType() != null
+                && RANGE_HEARING_DATE_TYPE.equals(listingData.getHearingDateType())) {
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:375]: Part of the advantage of using String.equals is that you don't need the preceding null check

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java [hunk 3]
```diff
@@ -366,7 +367,12 @@ private static StringBuilder getCaseCauseListByRoom(ListingData listingData, Str
     public static StringBuilder getListingDate(ListingData listingData) {
         var sb = new StringBuilder();
         if (listingData.getHearingDateType() != null
-                && listingData.getHearingDateType().equals(RANGE_HEARING_DATE_TYPE)) {
+                && RANGE_HEARING_DATE_TYPE.equals(listingData.getHearingDateType())
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:370]: Part of the advantage of using String.equals is that you don't need the preceding null check

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java [hunk 4]
```diff
@@ -384,4 +394,126 @@ private static StringBuilder getLiveCaseLoadReport(ListingData listingData) {
         return sb;
     }
 
+    private static StringBuilder getServedClaimsReport(ListingData listingData) {
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:397]: I think all the logic for this report deserves to be in its own class.

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java (hunk 1 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java:36]: right. I fix it.

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java (hunk 4 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:397]: can we do that as an improvement in another iteration?

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:397]: I mean after releasing this.

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java (hunk 3 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:370]: noted

**Review by jthmcts — APPROVED** [review]

**Pull request merged**